doc/developer/conventions.md

```diff
@@ -74,7 +75,6 @@ use `Scalar()` or `_ra` suffix when defining numbers from literals (e.g. `auto a
 
 * Constructors should be trival. All complex work goes in an `init()` function
```

Comment:
Out of scope of this PR but this should be avoided (https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c41-a-constructor-should-create-a-fully-initialized-object)

Reply:
Out of scope right, maybe simply do another pr